tests/framework/e2e/cluster.go

```diff
@@ -216,7 +219,7 @@ func InitEtcdProcessCluster(t testing.TB, cfg *EtcdProcessClusterConfig) (*EtcdP
 	etcdCfgs := cfg.EtcdAllServerProcessConfigs(t)
 	epc := &EtcdProcessCluster{
 		Cfg:     cfg,
-		lg:      zaptest.NewLogger(t),
+		lg:      cfg.Logger,
```

Comment:
unrelated change?

Reply:
seems to be a typo found during troubleshooting, can remove if you think it shouldn't be here.